commit 21fd080d18f280c19fdc5489726dcd6c66eb5fbf
Author: Gary Benson <gbenson@redhat.com>
Date:   Tue Jan 8 12:12:14 2013 +0000

    Also stop on "map_failed" where appropriate

Removed a comment patch hunk.

diff --git a/gdb/solib-svr4.c b/gdb/solib-svr4.c
index 5eb84ba..a46fd74 100644
--- a/gdb/solib-svr4.c
+++ b/gdb/solib-svr4.c
@@ -160,6 +160,7 @@ static const struct probe_info probe_info[] =
   { "init_start", NAMESPACE_NO_ACTION },
   { "init_complete", NAMESPACE_RELOAD },
   { "map_start", NAMESPACE_NO_ACTION },
+  { "map_failed", NAMESPACE_NO_ACTION },
   { "reloc_complete", NAMESPACE_UPDATE_OR_RELOAD },
   { "unmap_start", NAMESPACE_NO_ACTION },
   { "unmap_complete", NAMESPACE_RELOAD },
@@ -2056,6 +2058,9 @@ svr4_create_solib_event_breakpoints (struct gdbarch *gdbarch,
 	      info->probes[i] = find_probes_in_objfile (os->objfile, "rtld",
 							name);
 
+	      if (!strcmp (name, "rtld_map_failed"))
+		continue;
+
 	      if (!VEC_length (probe_p, info->probes[i]))
 		{
 		  free_probes (info);
